web/src/user_profile.js

```diff
@@ -371,7 +371,7 @@ function handle_remove_stream_subscription(target_user_id, sub, success, failure
     }
 }
 
-export function register_click_handlers() {
+function register_click_handlers() {
     $("body").on("click", ".info_popover_actions .view_full_user_profile", (e) => {
```

Comment:
So broadly I think there are two types of click handlers in this module. This one is a way to open the user profile widget, and in my view, belongs in the module responsible for the component that you click on to open this -- namely the user cards module. The rest of the click handlers in this function belong here; they are used to implement UI within this widget.

Reply:
The same seems to apply to a bunch of the others too.